bear/main.py.in

```diff
@@ -262,7 +262,11 @@ def parse_exec_trace(filename):
 
     logging.debug('parse exec trace file: %s', filename)
     with open(filename, 'r') as handler:
-        entry = json.load(handler)
+        try:
+            entry = json.load(handler)
+        except ValueError:
+            logging.debug('unable to parse: %s', filename)
+            return Execution(pid='', cwd='', cmd='')
```

Comment:
even better would be not to execute this given function if the file was empty.  Something strange I found was that problematic files seemed empty, but had a non zero length (st_size).  Maybe this had to do with calling read() after a json.load() without reseeking the input.